### ISSUE TITLE
Exception in DurableOrchestrationClient.GetStatusAsync() method

### ISSUE THREAD

**Issue by kijujjav** [Azure/azure-functions-durable-extension#688]
We have the below method to get the status of all workflows
 
```csharp
[FunctionName("QueryAllWorkflows")]
public static async Task<HttpResponseMessage> QueryAllWorkflows(
    [HttpTrigger(AuthorizationLevel.Function, methods: "get", Route = "workflow/status/all")] HttpRequestMessage queryWorkflowsRequest,
    [OrchestrationClient] DurableOrchestrationClient query,
    ILogger log)
{
    IList<DurableOrchestrationStatus> workflowInstances = await query.GetStatusAsync(); // there is an exception at this line
    List<WorkflowStatus> workflows = new List<WorkflowStatus>();
    // ...
}
 ```
 
**Exception:**
 ```
Executed 'QueryAllWorkflows' (Failed, Id=4ad18f57-293c-438f-ad1c-e3f6db291fda)
[4/2/2019 6:18:20 PM] System.Private.CoreLib: Exception while executing function: QueryAllWorkflows. DurableTask.AzureStorage:  is not a valid OrchestrationStatus value.
[4/2/2019 6:18:20 PM] Executed HTTP request: {
[4/2/2019 6:18:20 PM]   "requestId": "8fb5dc50-446f-430c-8367-8c8a9ccbcaf2",
[4/2/2019 6:18:21 PM]   "method": "GET",
[4/2/2019 6:18:21 PM]   "uri": "/workflow/status/all",
[4/2/2019 6:18:21 PM]   "identities": [
[4/2/2019 6:18:21 PM]     {
[4/2/2019 6:18:21 PM]       "type": "WebJobsAuthLevel",
[4/2/2019 6:18:21 PM]       "level": "Admin"
[4/2/2019 6:18:21 PM]     }
[4/2/2019 6:18:21 PM]   ],
[4/2/2019 6:18:21 PM]   "status": 500,
[4/2/2019 6:18:21 PM]   "duration": 4916
[4/2/2019 6:18:21 PM] }
```

**Full Stack Trace:**
```
[4/2/2019 7:18:43 PM] System.ArgumentException:  is not a valid OrchestrationStatus value.
[4/2/2019 7:18:43 PM]    at DurableTask.AzureStorage.Tracking.AzureTableTrackingStore.ConvertFromAsync(OrchestrationInstanceStatus orchestrationInstanceStatus, String instanceId) in C:\source\durabletask\src\DurableTask.AzureStorage\Tracking\AzureTableTrackingStore.cs:line 498
[4/2/2019 7:18:43 PM]    at DurableTask.AzureStorage.Tracking.AzureTableTrackingStore.<QueryStateAsync>b__43_0(OrchestrationInstanceStatus status) in C:\source\durabletask\src\DurableTask.AzureStorage\Tracking\AzureTableTrackingStore.cs:line 588
[4/2/2019 7:18:43 PM]    at System.Linq.Enumerable.SelectEnumerableIterator`2.MoveNext()
[4/2/2019 7:18:43 PM]    at System.Collections.Generic.List`1.AddEnumerable(IEnumerable`1 enumerable)
[4/2/2019 7:18:43 PM]    at System.Collections.Generic.List`1.InsertRange(Int32 index, IEnumerable`1 collection)
[4/2/2019 7:18:43 PM]    at DurableTask.AzureStorage.Tracking.AzureTableTrackingStore.QueryStateAsync(TableQuery`1 query, CancellationToken cancellationToken) in C:\source\durabletask\src\DurableTask.AzureStorage\Tracking\AzureTableTrackingStore.cs:line 589
[4/2/2019 7:18:43 PM]    at DurableTask.AzureStorage.AzureStorageOrchestrationService.GetOrchestrationStateAsync(CancellationToken cancellationToken) in C:\source\durabletask\src\DurableTask.AzureStorage\AzureStorageOrchestrationService.cs:line 1322
[4/2/2019 7:18:43 PM]    at Microsoft.Azure.WebJobs.DurableOrchestrationClient.GetStatusAsync(CancellationToken cancellationToken) in C:\GitHub\azure-functions-durable-extension\src\WebJobs.Extensions.DurableTask\DurableOrchestrationClient.cs:line 204
[4/2/2019 7:18:43 PM]    at NetAssistStatefulWorkflow.Actions.WorkflowQueries.QueryAllWorkflows(HttpRequestMessage queryWorkflowsRequest, DurableOrchestrationClient query, ILogger log) in D:\repos\Networking\NetAssist\src\NetAssistStatefulWorkflow\NetAssistStatefulWorkflow\Actions\WorkflowQueries.cs:line 144
```
 
**Http Request:**
 
GET http://localhost:7071/workflow/status/all

**Comment by cgillum** [comment]
Thanks for these details. I'll look into it.

**Comment by cgillum** [comment]
This fix has been released in the [v1.8.1 release](https://github.com/Azure/azure-functions-durable-extension/releases/tag/v1.8.1).
